src/analysis/liveness-transfer-function.h

```diff
@@ -47,38 +47,35 @@ class LivenessTransferFunction : public Visitor<LivenessTransferFunction> {
   // run before earlier CFG blocks. This improves performance by
   // reducing the number of state propagations needed, since we are
   // naturally following the backward flow at the beginning.
-  void enqueueWorklist(
-    const std::vector<BlockState<FinitePowersetLattice>>& stateBlocks,
-    std::queue<Index>& worklist) {
-    for (auto it = stateBlocks.rbegin(); it != stateBlocks.rend(); ++it) {
-      worklist.push((*it).getCFGBlock()->getIndex());
+  void enqueueWorklist(CFG* cfg, std::queue<Index>& worklist) {
+    for (auto it = cfg->rbegin(); it != cfg->rend(); ++it) {
+      worklist.push((*it).getIndex());
```

Comment:
It will be more forward compatible with interprocedural analyses to enqueue `BasicBlock*` rather than indices.